content/news/2015/10/2015-10-26-nihdigital-summit-optimizing-digital-to-reach-patients-scientists-clinicians-and-the-public.md

```diff
@@ -13,7 +13,6 @@ topics:
   - national-cancer-institute
   - national-institutes-of-health
   - NCI
-  - nih
   - US Department of Agriculture
```

Comment:
```suggestion
  - open-government
```

Reply:
This has product-management (changing to product-and-project-management ?) and social-media as topics; just leave these two
Open gov't is more about making gov't data/info available to the public